src/apps/collectionsUi/main.js

```diff
@@ -23,6 +24,7 @@ ReactDOM.render(
     <ConnectedRouter history={config.routing}>
       <I18nProvider>
         <Switch>
+          <Route component={Docs} path="/docs" />
```

Comment:
I'd say Docs should be after App because there's no reason for it to match before /app. when I see this I wonder if there's a reason that it needs to match first :-)

Reply:
sure